src/pstext.c

```diff
@@ -356,6 +368,8 @@ static int usage (struct GMTAPI_CTRL *API, int level) {
 	GMT_Message (API, GMT_TIME_NONE, "\t-N Do Not clip text that exceeds the map boundaries [Default will clip].\n");
 	GMT_Option (API, "O,P");
 	GMT_Message (API, GMT_TIME_NONE, "\t-Q For all text to be (l)lower or (u)pper-case [Default leaves text as is].\n");
+	GMT_Message (API, GMT_TIME_NONE, "\t-S Plot a shadow behind the text box.  Requires -G<color> to be given as well.\n");
```

Comment:
If it requires **-G** as well what is the **-S**_shade_ parameter used to?

Reply:
Text box may be white, yellow, or whatever, but the shade it casts need to be different for most cases.  See plot above.